### PR TITLE
Kolya's part(actual)

### PR DESCRIPTION
**Pull request by Nick-Shcherban**
added my part (mobile version of hiring block) to Ilya's footer.
made some changes in `body` - thus should be verified by other team members

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,10 +10,37 @@
     <link rel="stylesheet" href="styles/styles.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@300;600&display=swap" rel="stylesheet">
+    <!-- <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@300;600&display=swap" rel="stylesheet"> я добавил еще несколько размеров нужно нам шрифта, ссылка внизу -->
```
Comment (velmyk) [inline index.html]: Remove unused code

File: styles/styles.css [hunk 2]
```diff
@@ -1,3 +1,96 @@
...
+    box-sizing: border-box;
+    margin: 0;
+    font-family: 'Work Sans', 'sans-serif';
+}
+
+.hiring {
+    display: flex;
+    flex-direction: column;
+    width: 320px;
+    height: 866px;
```
Comment (velmyk) [inline styles/styles.css]: Are you sure to set strict height?

File: styles/styles.css [hunk 3]
```diff
@@ -1,3 +1,96 @@
...
+    font-size: 16px;
+    line-height: 22px;
+    margin: 7px 71px 19px 20px;
+    text-align: left;
+}
+.vacancies_action-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
```
Comment (velmyk) [inline styles/styles.css]: Do you need an `absolute` position here?

**Review by velmyk — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Nick-Shcherban) [reply index.html]: done

File: styles/styles.css (hunk 2 above)
Comment (Nick-Shcherban) [reply styles/styles.css]: done

File: styles/styles.css (hunk 3 above)
Comment (Nick-Shcherban) [reply styles/styles.css]: rebuild this in the next commit. please, verify

**Pull request merged**